source/read/change-streams.txt

```diff
@@ -0,0 +1,268 @@
+
+   * - ``startAtOperationTime``
+     - | Instructs the change stream to only provide changes that occurred at or after
+         the specified timestamp. 
+       | This option is mutually exclusive with ``startAfter`` and ``resumeAfter``.
+
+   * - ``collation``
+     - | Sets the collation to use for the change stream cursor.
+
+For a full list of ``watch()`` options, see `MongoDB\\Collection::watch()
+<{+api+}/method/MongoDBCollection-watch/>`__ in the API
+documentation.
+
+.. _php-change-stream-pre-post-image:
+
+Include Pre-Images and Post-Images
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important::
+
+   You can enable pre-images and post-images on collections only if your
+   deployment uses MongoDB v6.0 or later.
+
+By default, when you perform an operation on a collection, the
+corresponding change event includes only the delta of the fields
+modified by that operation. To see the full document before or after a
+change, specify the ``fullDocumentBeforeChange`` or the ``fullDocument``
+options in an array parameter to ``watch()``.
+
+The **pre-image** is the full version of a document *before* a change. To include the
+pre-image in the change stream event, set the ``fullDocumentBeforeChange`` option
+to one of the following values:
+
+- ``MongoDB\Operation\Watch::FULL_DOCUMENT_BEFORE_CHANGE_WHEN_AVAILABLE``: The change event includes
+  a pre-image of the modified document for change events. If the pre-image is not available, this
+  change event field has a ``null`` value.
+- ``MongoDB\Operation\Watch::FULL_DOCUMENT_BEFORE_CHANGE_REQUIRED``: The change event includes a pre-image
+  of the modified document for change events. If the pre-image is not available, the
+  {+php-library+} raises an error.  
+
+The **post-image** is the full version of a document *after* a change. To include the
+post-image in the change stream event, set the ``fullDocument`` option to
+one of the following values:
+
+- ``MongoDB\Operation\Watch::FULL_DOCUMENT_UPDATE_LOOKUP``: The change event includes a
+  copy of the entire changed document from some time after the change.
+- ``MongoDB\Operation\Watch::FULL_DOCUMENT_WHEN_AVAILABLE``: The change event includes
+  a post-image of the modified document for change events. If the post-image is not
+  available, this change event field has a ``null`` value.
+- ``MongoDB\Operation\Watch::FULL_DOCUMENT_REQUIRED``: The change event includes a post-image
+  of the modified document for change events. If the post-image is not available, the
+  server raises an error.
+
+The following example calls the ``watch()`` method on a collection and includes the post-image
+of updated documents by setting the ``fullDocument`` option:
+
+.. literalinclude:: /includes/read/change-streams.php
+   :start-after: start-change-stream-post-image
+   :end-before: end-change-stream-post-image
+   :language: php
+   :dedent:
+
+With the change stream application running, updating a document in the
```

Comment:
I think "updating" should be "update".

Per my earlier comment about pointing users away from using a web server, the following might be clearer:

> With the change stream application running in a separate shell, update a document...

Reply:
Added "in a separate shell" - "using" makes more grammatical sense in this case

Reply:
> "updating" makes more grammatical sense in this case

My mistake. I missed that the sentence continued with "prints a change event..." after the `:ref:` link.